packages/next/src/client/components/react-dev-overlay/ui/components/errors/dev-tools-indicator/dev-tools-indicator.tsx

```diff
@@ -73,13 +70,25 @@ interface C {
 
 const Context = createContext({} as C)
 
+function getInitialPosition() {
+  if (
+    typeof localStorage !== 'undefined' &&
+    localStorage.getItem('nextjs-dev-tools-position')
```

Comment:
feel this key is not special enough, maybe give a `__` prefix to indicate it's private